internal/backend/sessions/sessionworkflows/callbacks_signals.go

```diff
@@ -93,10 +98,16 @@ func (w *sessionWorkflow) nextSignal(wctx workflow.Context) func(context.Context
 			return nil, wctx.Err()
 		}
 
-		if signal == nil {
+		if signal == nil || signal.Name == "" {
 			return nil, nil
 		}
 
+		if sid, err := sdktypes.ParseSessionID(signal.Name); err == nil {
```

Comment:
How come lightweight wrappers of Temporal workflows don't needs this? (i.e. why only regular AK sessions?)